incubator/jenkins/values.yaml

```diff
@@ -0,0 +1,31 @@
+# Default values for jenkins.
+# This is a YAML-formatted file.
+# Declare name/value pairs to be passed into your templates.
+# name: value
+
+Master:
+  Name: jenkins-master
+  Image: "gcr.io/kubernetes-charts-ci/jenkins-master-k8s"
+  ImageTag: "v6"
+  ImagePullPolicy: "Always"
+  Component: "jenkins-master"
+  AdminUser: admin
+# AdminPassword: <defaults to random>
+  Cpu: "200m"
```

Comment:
Added a value so that this can be configured as necessary. Defaults to nothing as its hard to make a good recommendation, as you stated.